system/lib/wasmfs/backends/fetch_backend.cpp

```diff
@@ -13,15 +13,17 @@
 // See library_wasmfs_fetch.js
 
 extern "C" {
-void _wasmfs_create_fetch_backend_js(wasmfs::backend_t);
+void _wasmfs_create_fetch_backend_js(wasmfs::backend_t, char*);
 }
 
 namespace wasmfs {
 
 extern "C" backend_t wasmfs_create_fetch_backend(char* base_url) {
   // TODO: use base url, cache on JS side
```

Comment:
This stale comment should be deleted.

Reply:
Ok. What about the "cache on JS side" part? I'm not doing it in this PR.